packages/components/radio/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@radix-ui/react-radio-group": "1.1.2",
+    "@radix-ui/react-label": "2.0.1",
```

Comment:
IMO we could create a simple `Label` component

Reply:
agree

Reply:
there could be some kind of custom brand elements that might be labeled. We should provide a spark solution for them